RocketChat/Rocket.Chat
# fix: team channel reload error message

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce

1. create a team - A
2. create a channel - B.
3. Add existing channel B to team A.
4. Red toast bar appears
## Further comments

[TC-432]

[TC-432]: https://rocketchat.atlassian.net/browse/TC-432?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
